src/actions/sendToken.ts

```diff
@@ -0,0 +1,77 @@
+import { PublicKey } from '@solana/web3.js';
+import { ASSOCIATED_TOKEN_PROGRAM_ID, Token, TOKEN_PROGRAM_ID, u64 } from '@solana/spl-token';
+import { Wallet } from '../wallet';
+import { Connection } from '../Connection';
+import { sendTransaction } from './transactions';
+import { Transaction } from '../Transaction';
+import { Account } from '../Account';
+import { CreateAssociatedTokenAccount } from '../programs/shared/transactions/CreateAssociatedTokenAccount';
+
+interface ISendTokenParams {
+  connection: Connection;
+  wallet: Wallet;
+  // token account address
+  source: PublicKey;
+  // destination wallet address
+  destination: PublicKey;
+  mint: PublicKey;
+  amount: number | u64;
```

Comment:
What about having a BN here?

Reply:
`u64` extends `BN`
And to be honest I don't have any idea how can we send more than 1 token from a token account with NFT